src/Figure/Triangle.java

```diff
@@ -0,0 +1,45 @@
+package Figure;
+
+public class Triangle implements Area {
+    private double a; // довжина першой сторони
```

Comment:
remove all comments from this class!
You should not give a name like 'a' or 'b' or 'c' to your variables! Rename all of them to something more meaningful.